server/game/cards/01_SOR/TieAdvanced.ts

```diff
@@ -12,7 +12,7 @@ export default class TieAdvanced extends NonLeaderUnitCard {
 
     public override setupCardAbilities() {
         this.addWhenPlayedAbility({
-            title: 'Give 2 experience tokens to a friendly Imperial unit',
+            title: 'Give 2 Experience tokens to a friendly IMPERIAL unit',
```

Comment:
I would revert IMPERIAL to Imperial here

Reply:
will do